services/core/shared/indexdb/mysql.js

```diff
@@ -214,6 +214,7 @@ const getDbInstance = async (tableName, tableConfig, connEndpoint = config.endpo
 		if (params.sort) {
 			const [sortProp, sortOrder] = params.sort.split(':');
 			query.orderBy(sortProp, sortOrder);
+			query.whereNotNull(sortProp);
```

Comment:
It works well but returns incorrect count. Now, we need to add given logic to our count function as well to get exact count: 
```
if (params.sort) {
  const [sortProp, sortOrder] = params.sort.split(':');
  query.orderBy(sortProp, sortOrder);
  query.whereNotNull(sortProp);
}
```